JoyboyCommunity/index.js

```diff
@@ -1,5 +1,5 @@
 import registerRootComponent from "expo/build/launch/registerRootComponent";
 import "fast-text-encoding";
-import Wrapper from "./src/bootstrap/Wrapper";
+import App from "./src/app/App";
 
-registerRootComponent(Wrapper);
```

Comment:
i personally feel it is easier to track wrappers like sentry and themeing in one file so named it wrapper and this can also include react context providers. Any reason this was changed to be called provider since this means it mainly involves react context providers?

Reply:
I named it to providers because it only contains react context providers, so it will be easier to track context providers and add new. There's also no harm in being Wrapper. I can make it a Wrapper instead if would you like.

Reply:
Aah no i meant if we add sentry or any other util layer then where would we plan to add it?

Reply:
Not heavily related to react context too so can be flexible with use

Reply:
Yeah, sounds good! Changed it to Wrapper.